be/src/exec/pipeline/exchange/shuffler.h

```diff
@@ -75,8 +75,13 @@ class Shuffler {
             if constexpr (!two_level_shuffle) {
                 shuffle_id = channel_id;
             } else {
-                uint32_t driver_sequence = ReduceOp()(HashUtil::xorshift32(hash_values[i]), _num_shuffles_per_channel);
-                shuffle_id = channel_id * _num_shuffles_per_channel + driver_sequence;
+                if (_num_shuffles_per_channel == 1) {
```

Comment:
prefer move to outer loop whether it will break the vectorized

Reply:
this file is no need to modify.